stream-chat-android-ui-components/src/main/res/values/styles.xml

```diff
@@ -337,6 +337,48 @@
         <item name="android:layout_marginBottom">12dp</item>
     </style>
 
+    <style name="StreamUi.ChannelListView" parent="StreamUi">
```

Comment:
Is this being used? I think it should go inside `StreamUiTheme`.

Reply:
It's used [here](https://github.com/GetStream/stream-chat-android/pull/2064/files#diff-1d31aa89c0698cf181975aecd03f3f8cd8d448fd58e52d1ae2cd7376aec99391R76) so we don't need to add it to `StreamUiTheme`